__tests__/app.test.js

```diff
@@ -9,6 +9,38 @@ afterAll(() => db.end())
 //#endregion
 
 describe('Model: Topics', () => {
+    describe('/api', () => {
+        test('GET 200   | Returns 200 status when passed', () => {
+            return request(app).get('/api').expect(200)
+        });
+        test('GET 200   | Returns an object with endpoints within', () => {
+            return request(app).get('/api').expect(200).then(({body}) => {
+                expect(Object.keys(body).length > 0).toBe(true)
+                for (const key in body) {
+                    expect(body[key]).toHaveProperty("description")
+                    expect(body[key]).toHaveProperty("queries")
+                    expect(body[key]).toHaveProperty("bodyFormat")
+                    expect(body[key]).toHaveProperty("exampleResponse")
+                }
+            })
+        });
+        test('GET 200   | Returns instructions for all available endpoints', () => {
```

Comment:
On the surface, I'm not sure what this test is doing differently to the one above?

Reply:
It seems to just be checking that all the endpoints are accounted for. We could do this by asserting with toEqual against the endpoints file itself